bindings/python/ML/CMakeLists.txt

```diff
@@ -8,10 +8,14 @@ if(TARGET BipedalLocomotion::ML)
 
   add_bipedal_locomotion_python_module(
     NAME MLBindings
-    SOURCES src/Module.cpp src/MANN.cpp src/MANNAutoregressive.cpp
-            src/MANNTrajectoryGenerator.cpp src/MANNAutoregressiveInputBuilder.cpp
-    HEADERS ${H_PREFIX}/Module.h ${H_PREFIX}/MANN.h ${H_PREFIX}/MANNAutoregressive.h
-            ${H_PREFIX}/MANNTrajectoryGenerator.h ${H_PREFIX}/MANNAutoregressiveInputBuilder.h
+    SOURCES src/Module.cpp src/MANN.cpp src/velMANN.cpp
+            src/MANNAutoregressive.cpp src/velMANNAutoregressive.cpp
+            src/MANNTrajectoryGenerator.cpp src/velMANNTrajectoryGenerator.cpp
+            src/MANNAutoregressiveInputBuilder.cpp src/velMANNAutoregressiveInputBuilder.cpp
+    HEADERS ${H_PREFIX}/Module.h ${H_PREFIX}/MANN.h ${H_PREFIX}/velMANN.h
+            ${H_PREFIX}/MANNAutoregressive.h ${H_PREFIX}/velMANNAutoregressive.h
```

Comment:
We may call the file/classes PIMANN etc etc

Reply:
Based on f2f conversation, leaving as is. Explanation: the difference in implementation is due only to the new velocity-based features, the PI is irrelevant here.